### PR TITLE
feat(orchestrator): display a description modal before triggering infra-wfs that resulted from an assessment wf

### PR DESCRIPTION
**Pull request by yzhao583**
Jira: [https://issues.redhat.com/browse/FLPATH-924](https://issues.redhat.com/browse/FLPATH-924)

Display a description modal before triggering infra-wfs that resulted from an assessment wf

image

### CODE REVIEW

**Review by jkilzi — CHANGES_REQUESTED** [review]
PTAL at the comments

File: plugins/orchestrator/src/components/WorkflowDescriptionModal.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          >
+            <CloseIcon />
+          </IconButton>
+        </Box>
+      </DialogTitle>
+      <DialogContent>
+        <Box>{workflow.description}</Box>
+        <Box>
+          <p>Are you sure you want to run this workflow?</p>
+        </Box>
```
[the remark this answers is not in the document]
Comment (yzhao583) [reply plugins/orchestrator/src/components/WorkflowDescriptionModal.tsx:74]: @jkilzi This is required by UXD, PTAL at [design](https://www.figma.com/proto/XjdTCb4vDQ7L4Jz1TdHKGq/RHDH---Workflow-Orchestrator?page-id=0%3A1&type=design&node-id=539-1418&viewport=3193%2C-4624%2C0.5&t=OuIn8Kcuq53W4KPT-1&scaling=min-zoom&starting-point-node-id=848%3A1670), if you click the `Move 2 Kube` link, you should be able to see the modal. Thanks

[the remark this answers is not in the document]
Comment (jkilzi) [reply plugins/orchestrator/src/components/WorkflowDescriptionModal.tsx:74]: ack, I left the same question in the design, let's wait (not too much) for an explanation.
cc @nfarkas

[the remark this answers is not in the document]
Comment (yzhao583) [reply plugins/orchestrator/src/components/WorkflowDescriptionModal.tsx:74]: Ok

**Comment by yzhao583** [comment]
@jkilzi @nfarkas Can we TAL at this? Thanks

File: plugins/orchestrator/src/components/WorkflowDescriptionModal.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (caponetto) [reply plugins/orchestrator/src/components/WorkflowDescriptionModal.tsx:74]: I'd go with whatever we have on the design and open a new ticket if necessary.

[the remark this answers is not in the document]
Comment (yzhao583) [reply plugins/orchestrator/src/components/WorkflowDescriptionModal.tsx]: @jkilzi Modified the PR based on the new change from UXD, can you PTAL? thanks

**Review by jkilzi — APPROVED** [review]

**Pull request merged**